src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let join = '';
```

Comment:
It's better to name it `resultString` for example or smth like that.
Because `join` mostly uses as a verb for function names.